python/paddle/nn/functional/flash_attention.py

```diff
@@ -995,6 +995,7 @@ def scaled_dot_product_attention(
     attn_mask: Tensor | None = None,
     dropout_p: float = 0.0,
     is_causal: bool = False,
+    scaling_factor: float = None,
```

Comment:
Adding parameters is usually placed at the end of function to avoid compatibility issues

Reply:
Done